### ISSUE TITLE
Automatically delete ALL messages/files

### ISSUE THREAD

**Issue by svyatov** [mgleon08/slack_neuralyzer#1]
Right now there is no way to automatically delete all the files/messages across all the channels (-C |  -D | -G | -M). Is this Slack API limitation?

**Comment by mgleon08** [comment]
Yes, this Is Slack API limitation. Slack must to specified channel.
